source/front-end/githubAsDatabase.js

```diff
@@ -107,9 +107,9 @@ export default {
                 })
 
                 promisesToWait.push(exerciceDirP)
-
-                return Promise.all(promisesToWait).then(() => opérationsHautNiveauByYear)
```

Comment:
ce `return` était à l'intérieur d'une boucle for-of et donc ne résolvait qu'un seul fichier

Il suffisait de le sortir de la boucle pour résoudre le problème